schema/extensions/ratelimiting.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$id": "https://serverlessworkflow.io/schemas/0.7/extensions/ratelimiting.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Serverless Workflow specification - Various workflow rate limiting settings",
+  "type": "object",
+  "definitions": {
+    "ratelimiting": {
+      "type": "object",
+      "description": "Serverless Workflow Rate Limiting Extension",
+      "properties": {
+        "extensionid": {
+          "type": "string",
+          "default": "workflow-ratelimiting-extension",
+          "description": "Extension unique identifier"
+        },
+        "workflowid": {
+          "type": "string",
+          "minLength": 1,
+          "description": "Workflow definition unique identifier (workflow id property)"
+        },
+        "workflowVersions": {
+          "type": "array",
+          "description": "Workflow versions. If not defined, applies to all workflow instances (regardless of their associated workflow version)",
+          "items": {
+            "type": "string"
+          }
+        },
+        "singleInstance": {
+          "description": "Rate Limit settings per single instance of a workflow with provided workflowid",
+          "$ref": "#/definitions/ratelimits"
+        },
+        "allInstances": {
+          "description": "Rate Limit settings across all instance of a workflow with provided workflowid",
+          "$ref": "#/definitions/ratelimits"
+        }
+      },
+      "required": [
+        "extensionid",
+        "workflowid"
+      ]
+    },
+    "ratelimits": {
+      "type": "object",
+      "properties": {
+        "maxActionsPerSecond": {
+          "type": "number",
+          "default": 0,
+          "description": "Sets the rate limiting on number of actions that can be executed per second. Notice that the number is represented as number type, so that you can set it to less than 1 if needed. For example, set the number to 0.1 means workflow actions can be executed once every 10 seconds. Default zero value means 'unlimited'"
```

Comment:
```suggestion
          "description": "Sets the rate-limiting on a number of actions that can be executed per second. Notice that the number is represented as number type so that you can set it to less than 1 if needed. For example, set the number to 0.1 means workflow actions can be executed once every 10 seconds. Default zero value means 'unlimited'"
```